make/autoconf/jdk-options.m4

```diff
@@ -638,14 +638,11 @@ AC_DEFUN([JDKOPT_EXCLUDE_TRANSLATIONS],
 
 ################################################################################
 #
-# Optionally disable man pages
+# Optionally disable man pages (deprecated)
```

Comment:
Elsewhere in JDK "deprecated" means "still works but don't use it", whereas here (and perhaps in build system overall?) it means "ignored but doesn't cause an error".

Reply:
Yeah, you are right. I actually had the same thought when changing this. I don't know how this came to be. Obviously, I am to blame, since I created the `UTIL_DEPRECATED_ARG_ENABLE` function. :-) What I mean is, I don't really understand why I thought "deprecated" was a good way to describe this.

I opened https://bugs.openjdk.org/browse/JDK-8343817 to deal with this systematically in the entire build system. Are you ok with keeping this in line with the rest of the build system for now?

Reply:
Sure - no problem. Thanks for doing the follow up.